include/SDL3_image/SDL_image.h

```diff
@@ -2337,6 +2337,43 @@ extern SDL_DECLSPEC bool SDLCALL IMG_CloseAnimationStream(IMG_AnimationStream *s
  */
 extern SDL_DECLSPEC IMG_Animation *SDLCALL IMG_LoadAPNGAnimation_IO(SDL_IOStream *src);
 
+/**
```

Comment:
Can you move these in the header up above the JPG saving functions? Ideally the save functions would be grouped together and alphabetically, and I can update that later.